yarn.lock

```diff
@@ -1239,6 +1239,11 @@ acorn@^6.0.7:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.3.0.tgz#0087509119ffa4fc0a0041d1e93a417e68cb856e"
   integrity sha512-/czfa8BwS88b9gWQVhc8eknunSA2DoJpJyTQkhheIf5E48u1N0R4q/YxxsAeqRrmK9TQ/uYfgLDfZo91UlANIA==
 
+acorn@^7.0.0:
```

Comment:
Why did this file change in this PR? I don't see any package/dependency changes?

Reply:
Oh that is weird ... I seems to be some package updating.

I might surely delete `yarn.lock, `node_modules/` and run `yarn` between two branches.

That can explain it ?

Reply:
That might explain it. Deleting `yarn.lock` can be a bit dangerous - it sometimes has side effects. Generally I'd like to avoid changing `yarn.lock` if there are no `package.json` changes.